RELEASE/scripts/autoscend/combat/auto_combat_jarlsberg.ash

```diff
@@ -0,0 +1,795 @@
+//Path specific combat handling for Avatar of Jarlsberg
+
+string auto_jarlsbergCombatHandler(int round, monster enemy, string text)
+{
+	boolean blocked = contains_text(text, "(STUN RESISTED)");
+	int damageReceived = 0;
+	if (!isActuallyEd())
```

Comment:
you forgot to change this line. should be checking if we are jarlsberg not ed

Reply:
Yeah, this was very much a "get a draft up", it's nowhere near ready yet.